smarts/core/waymo_map.py

```diff
@@ -305,17 +310,28 @@ def _compute_lane_intersections(self) -> Dict[int, Set[int]]:
                         intersections[cand_id].add(feat_id)
                         break
 
-        # Remove "fake" incoming/outgoing lanes that aren't true intersections
+        # Remove lanes that aren't true intersections
         mappings_to_remove = []
         for feat_id, intersect_ids in intersections.items():
-            lane_pts = self._polyline_cache[feat_id][0]
+            lane_pts = np.array(self._polyline_cache[feat_id][0])
```

Comment:
Since we're also doing this for every lane above (in the loops where we set `line1` and `line2`), we may as well just save the result of doing this once somewhere (probably in the `Lane` class or in the `_polyline_cache` itself?) and not repeat the work.